toolkit-maven-plugin/src/main/java/org/apache/servicecomb/toolkit/plugin/GenerateUtil.java

```diff
@@ -81,10 +83,55 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
             .substring(0, file.toFile().getName().indexOf(".")));
 
         docGenerator.configure(docGeneratorConfig);
-        docGenerator.generate();
+        if (!docGenerator.generate()) {
+          throw new RuntimeException("failed to generate document by generator " + type);
```

Comment:
Can we add more specific information here? 
You can just throw the exception with detail information inside of generate method of docGenerator.